### PR TITLE
docs: docs updates. added section observability

### PR DESCRIPTION
**Pull request by Shearwater1978**

### CODE REVIEW

File: docs/Observability/2. prometheus.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Prometheus method
+## Introduction
+Prometheus is an open-source monitoring and alerting toolkit designed for reliability and scalability. It collects metrics from configured targets at specified intervals, stores them in a time-series database, and provides powerful querying capabilities. With its flexible architecture, Prometheus is particularly suited for dynamic environments, making it a popular choice for cloud-native applications and microservices. Its intuitive visualization tools help teams gain deep insights into system performance, ensuring timely detection of issues.
+## Configuration
+### AI DIAL Core Settings
+Add the following environment variable to AI DIAL Core configuration. Refer to [AI DIAL Core](https://github.com/epam/ai-dial-helm/tree/main/charts/dial-core) to learn more.
+```json
+containerPorts.metrics
```
Comment (alexey-ban) [inline docs/Observability/2. prometheus.md:8]: metrics.enabled
metrics.serviceMonitor.enabled

File: docs/Observability/2. prometheus.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+```
+### AI DIAL Extentions Settings
+Add the following environment variable to AI DIAL Extentions configuration. Refer to [AI DIAL Extentions](https://github.com/epam/ai-dial-helm/tree/main/charts/dial-extension) to learn more.
+```json
```
Comment (alexey-ban) [inline docs/Observability/2. prometheus.md:12]: metrics.enabled
metrics.serviceMonitor.enabled

The default port for collecting metrics in most dial applications is 9464. If necessary, you can change the parameter. `containerPorts.metrics`

File: docs/Observability/3. otlp.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# OpenTelemetry method
+## Introduction
+OpenTelemetry is an open-source observability framework designed to standardize the collection of telemetry data across distributed systems. By providing a unified set of APIs, libraries, and agents, it enables developers to capture traces, metrics, and logs from their applications seamlessly. OpenTelemetry simplifies the monitoring process and enhances visibility into application performance and reliability, making it easier to troubleshoot issues and optimize systems in real-time.
```
Comment (alexey-ban) [inline docs/Observability/3. otlp.md:3]: Please add link to opentelemetry

File: docs/Observability/2. prometheus.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Prometheus method
```
Comment (alexey-ban) [inline docs/Observability/2. prometheus.md:1]: In this document, we describe all types of data. We need to assist the user with both metrics and logging configuration. Therefore, let's come up with section names for these two approaches and add details about log collection. Alternatively, we could add another document about logs.

File: docs/Observability/2. prometheus.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+## Introduction
+Prometheus is an open-source monitoring and alerting toolkit designed for reliability and scalability. It collects metrics from configured targets at specified intervals, stores them in a time-series database, and provides powerful querying capabilities. With its flexible architecture, Prometheus is particularly suited for dynamic environments, making it a popular choice for cloud-native applications and microservices. Its intuitive visualization tools help teams gain deep insights into system performance, ensuring timely detection of issues.
```
Comment (alexey-ban) [inline docs/Observability/2. prometheus.md:3]: link to prometheus and prometheus operator

File: docs/Observability/3. otlp.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+## Configuration
+### Configuration Guidelines
+    `TBD?`
+### Configure AI DIAL
+Environment Variables for the Configuration of Opentelemetry
+All standard env variables you could find in the official opentelemetry [documentation](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/logging/logging.html). If no value set for the OTEL_METRICS_EXPORTER then OpenTelemetry Prometheus Metric Exporter will be used. If value set to "otlp" the OpenTelemetry Collector Metrics Exporter for web and node will be used.
```
Comment (alexey-ban) [inline docs/Observability/3. otlp.md:9]: The DIAL ecosystem includes other programming languages (for example chat use nodejs )

File: docs/Observability/2. prometheus.md (hunk 1 above)
Comment (Shearwater1978) [reply docs/Observability/2. prometheus.md:10]: Done

File: docs/Observability/2. prometheus.md (hunk 2 above)
Comment (Shearwater1978) [reply docs/Observability/2. prometheus.md:18]: Done

File: docs/Observability/3. otlp.md (hunk 3 above)
Comment (Shearwater1978) [reply docs/Observability/3. otlp.md]: Done

File: docs/Observability/2. prometheus.md (hunk 4 above)
Comment (Shearwater1978) [reply docs/Observability/2. prometheus.md:1]: Done. Created the dedicated "topic"

File: docs/Observability/2. prometheus.md (hunk 5 above)
Comment (Shearwater1978) [reply docs/Observability/2. prometheus.md]: Done

File: docs/Observability/3. otlp.md (hunk 6 above)
Comment (Shearwater1978) [reply docs/Observability/3. otlp.md:7]: Not sure, but done

File: docs/Observability/Observability.md [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+
+These environment variables you can use to configure logging:
+
+```yaml
+AIDIAL_LOG_FILE: #Place when the log file should be stored.
+AIDIAL_LOG_LEVEL: #The logging levels used are ERROR, WARN, INFO, DEBUG, and TRACE.
+```
+
+#### Bedrock Adapter
+
```
Comment (sr-remsha) [inline docs/Observability/Observability.md:72]: Do we want to add some text here?

File: docs/Observability/Observability.md [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+```yaml
+LOG_LEVEL: #Level filter for the Adapter logger. Values: `TRACE, DEBUG, INFO, WARN, ERROR, FATAL`. Use `DEBUG` for dev purposes and INFO in prod. It is strongly recommended not to use the logging level `DEBUG` for prod purposes.
+AIDIAL_LOG_LEVEL: #AI DIAL SDK Level filter for the LLM and response logging. Values: `TRACE, DEBUG, INFO, WARNING, ERROR, FATAL`.
+```
+
+#### Vertex Adapter
+
```
Comment (sr-remsha) [inline docs/Observability/Observability.md:79]: Do we want to add some text here?

File: docs/Observability/Observability.md [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+```yaml
+LOG_LEVEL: #Level filter for the Adapter logger. Values: `TRACE, DEBUG, INFO, WARN, ERROR, FATAL`. Use `DEBUG` for dev purposes and INFO in prod. It is strongly recommended not to use the logging level `DEBUG` for prod purposes.
+AIDIAL_LOG_LEVEL: #AI DIAL SDK Level filter for the LLM and response logging. Values: `TRACE, DEBUG, INFO, WARNING, ERROR, FATAL`.
+```
+
+#### OpenAI Adapter
+
```
Comment (sr-remsha) [inline docs/Observability/Observability.md:86]: Do we want to add some text here?

File: docs/Observability/Observability.md [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+```yaml
+LOG_LEVEL: #Level filter for the Adapter logger. Values: `TRACE, DEBUG, INFO, WARN, ERROR, FATAL`. Use `DEBUG` for dev purposes and INFO in prod. It is strongly recommended not to use the logging level `DEBUG` for prod purposes.
+```
+
+#### DIAL Adapter
+
```
Comment (sr-remsha) [inline docs/Observability/Observability.md:92]: Do we want to add some text here?

File: docs/Observability/Observability.md [hunk 11]
```diff
@@ -0,0 +1,161 @@
...
+  OTEL_PYTHON_FASTAPI_EXCLUDED_URLS: "<exclude_url>" # to exclude certain URLs from tracking
+```
+
+#### Node.js Components
+
+* If the value for **OTEL_METRICS_EXPORTER** is not set, the [OpenTelemetry Prometheus Metric Exporter](https://www.npmjs.com/package/@opentelemetry/exporter-prometheus) will be used. 
+* If its value is set to `"otlp"`, the [OpenTelemetry Collector Metrics Exporter for Web and Node](https://www.npmjs.com/package/@opentelemetry/exporter-metrics-otlp-http) will be used.
+
+Example configuration of OpenTelemetry:
+
```
Comment (sr-remsha) [inline docs/Observability/Observability.md:155]: This is configuration for what? Core?

Comment (alexey-ban) [reply docs/Observability/Observability.md:155]: All components that use the python ai-dial-sdk library.

File: docs/Observability/Observability.md [hunk 12]
```diff
@@ -0,0 +1,161 @@
...
+#### Python Components
+
+All standard python environment variables you can find in the official OpenTelemetry [documentation](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/logging/logging.html). 
+
+* If the value for **OTEL_METRICS_EXPORTER** is not set, the [OpenTelemetry Prometheus Metric Exporter](https://www.npmjs.com/package/@opentelemetry/exporter-prometheus) will be used. 
+* If its value is set to `"otlp"`, the [OpenTelemetry Collector Metrics Exporter for Web and Node](https://www.npmjs.com/package/@opentelemetry/exporter-metrics-otlp-http) will be used.
+
+Example configuration of OpenTelemetry:
+
```
[the remark this answers is not in the document]
Comment (alexey-ban) [reply docs/Observability/Observability.md:138]: All components that use the python ai-dial-sdk library.

File: docs/Observability/Observability.md (hunk 9 above)
Comment (Shearwater1978) [reply docs/Observability/Observability.md:86]: Done

File: docs/Observability/Observability.md (hunk 10 above)
Comment (Shearwater1978) [reply docs/Observability/Observability.md:92]: Done

**Review by sr-remsha — DISMISSED** [review]

File: docs/Observability/Observability.md [hunk 13]
```diff
@@ -60,36 +60,44 @@ OTEL_LOG_LEVEL: #Log level used by the SDK logger.` Values: `TRACE, DEBUG, INFO,
 ```
 
 #### Core
+The main component of AI DIAL, which provides unified API to different chat completion and embedding models, assistants, and applications.
 
-These environment variables you can use to configure logging:
-
+The next environment variables you can for the configuring logging:
```
Comment (sr-remsha) [inline docs/Observability/Observability.md:65]: Something wrong here

**Review by sr-remsha — APPROVED** [review]

**Review by sr-remsha — APPROVED** [review]

**Pull request merged**